Project.toml

```diff
@@ -5,14 +5,18 @@ version = "0.1.0"
 
 [deps]
 AbstractAlgebra = "c3fe647b-3220-5bb0-a1ea-a7954cac585d"
+GAP = "c863536a-3901-11e9-33e7-d5cd0df7b904"
 Hecke = "3e1990a7-5d81-5526-99ce-9ba3ff248f21"
 Nemo = "2edaba10-b0f1-5616-af89-8c11ac63239a"
 Polymake = "d720cf60-89b5-51f5-aff5-213f193123e7"
+Singular = "bcd08a7b-43d2-5ff7-b6d4-c458787f915c"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [compat]
-julia = "1.0, 1.1, 1.2"
 AbstractAlgebra = "^0.8.0"
+GAP = "^0.2.3"
 Hecke = "^0.7.0"
```

Comment:
Can you replace this with `0.7.2`? I just tagged a new version which should fix one of the errors.